peer/test_utils.go

```diff
@@ -462,6 +464,32 @@ func (s *mockStatusMgr) RequestDisable(_ wire.OutPoint, _ bool) error {
 	return nil
 }
 
+type mockChainArb struct{}
+
+// newMockChainArb returns an instance of *mockChainArb.
+func newMockChainArb() *mockChainArb { return &mockChainArb{} }
```

Comment:
Another great candidate for the generic `testify/mock` :)

Reply:
I don't really see the point? Everything here returns default values - the mock would just assert that nil is returned?

Reply:
IMO can defer this to another PR